lib/mix/tasks/swagger.generate.ex

```diff
@@ -26,44 +27,46 @@ defmodule Mix.Tasks.Phx.Swagger.Generate do
   defp top_level_namespace, do: Mix.Project.get().application()[:mod] |> elem(0) |> Module.split |> Enum.drop(-1) |> Module.concat
   defp app_name, do: Mix.Project.get().project()[:app]
   defp default_swagger_file_path, do: app_path() <> "swagger.json"
-  defp default_router_module, do: Module.concat([top_level_namespace(), :Web, :Router])
-
+  defp default_router_module_name, do: Module.concat([top_level_namespace(), :Web, :Router])
+  defp default_endpoint_module_name, do: Module.concat([top_level_namespace(), :Web, :Endpoint])
+  defp router_module(switches), do: switches |> Keyword.get(:router, default_endpoint_module_name()) |> attempt_load
```

Comment:
Please add the `()` after `attempt_load` on these two lines.
I believe the convention in this repo is to use `()` everywhere.